src/client.js

```diff
@@ -13,7 +13,7 @@ class NgrokClient {
   constructor(processUrl) {
     this.internalApi = got.extend({
       prefixUrl: processUrl,
-      retry: 0,
+      retry: 3,
```

Comment:
With retries from `got` and retries based on the `isRetriable` function we have two different ways that retries might happen. Is there a reason to include retries from `got` specifically?

Reply:
Okay, makes sense to put this back to 0.